keepassxc-browser/content/autocomplete.js

```diff
@@ -132,9 +137,26 @@ class Autocomplete {
         // Update credentials to menu div
         for (const c of this.elements) {
             const item = document.createElement('div');
-            item.textContent = c.label;
+            item.classList.add('kpxcAutocomplete-item');
             item.setAttribute('uuid', c.uuid);
 
+            if (c.group !== null) {
```

Comment:
Is there some way we could use an html template instead of hard coding DOM?

Reply:
It should possible to use inject and put HTML files to `web_accesible_resources`. But I'm not sure how Manifest V3 will allow these. It must be also made sure that it doesn't break any Shadow DOM implementations. So in some way it's easier to hard-code DOM.

Reply:
OK, I don't want a more complicated solution to replace this so gtg.